### PR TITLE
Update Use OpenID Connect to Build a Simple Node.js Website

### PR DESCRIPTION
**Pull request by tanay1337**
Updates [Use OpenID Connect to Build a Simple Node.js Website](https://developer.okta.com/blog/2017/10/19/use-openid-connect-to-build-a-simple-node-website) to use the latest dependencies.

Updates to the GitHub repo: https://github.com/oktadev/okta-express-example/pull/3

### CODE REVIEW

**Review by alisaduncan — APPROVED** [review]
🍠

**Review by havi — COMMENTED** [review]
Hey Tanay, just wondering, why is the word wrap so weird here?

**Comment by tanay1337** [comment]
@havi I guess the original author wrote it in that style.

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 1]
```diff
@@ -10,127 +10,78 @@ communities: [javascript]
...
+last few years, authentication practices have changed quite a bit.
 
 Today I'm going to show you how to use OpenID Connect to build an extremely
 simple Node.js website (using Express.js) that allows you to manage your users,
 log them in, and log them out.
 
-Back in the day, all websites would require users to register with a
-username/password and log in with those same credentials. This was simple, but
-caused a lot of security problems because developers would need to write the
+All websites would require users to register with a
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
Websites used to require users to register with a
```
the current construction is over-complicated and a bit awkward...Also, I would avoid saying  "All"

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 2]
```diff
@@ -10,127 +10,78 @@ communities: [javascript]
...
+username/password and log in with those same credentials back in the day. This method was simple but
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
username/password and log in with those same credentials. This method was simple but
```
"back in the day" seems to dangle as a closing phrase, but I don't think it's necessary here.

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 3]
```diff
@@ -10,127 +10,78 @@ communities: [javascript]
...
+caused many security problems because developers would need to write the
 code to authenticate the user directly, store their credentials, manage their
-data, etc. It also required developers to build custom authorization schemes so
-that they could track what permissions their users had to perform certain
+data, etc. It also required developers to build custom authorization schemes to track what permissions their users had to perform certain
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
data, etc. This practice also required developers to build custom authorization schemes to track the user permissions necessary to perform certain 
```
simplified

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 4]
```diff
@@ -10,127 +10,78 @@ communities: [javascript]
...
+account with a large OAuth provider (Google, Facebook, etc.) and allow users to log in to your service via their OAuth account with that provider. This method had some excellent
+benefits: developers no longer needed to worry about storing passwords and managing
+credentials. The downside was that OAuth is a flexible protocol and doesn't lay
 out rules around authorization, data management, etc. This means that developers
-using pure OAuth are required to write a lot of custom security code themselves,
+using pure OAuth must write a lot of custom security code themselves,
 which causes problems.
 
-Just recently, however, [OpenID Connect](http://openid.net/connect/) (OIDC) has
+Then, [OpenID Connect](http://openid.net/connect/) (OIDC) has
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
Then, [OpenID Connect](http://openid.net/connect/) (OIDC) 
```
verb tense

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 5]
```diff
@@ -10,127 +10,78 @@ communities: [javascript]
...
 come onto the scene. It's a protocol built on top of OAuth that provides
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
came onto the scene. It's a protocol built on top of OAuth that provides
```

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 6]
```diff
@@ -335,10 +290,10 @@ now are the following two:
 
 - `cookie.httponly`: this option tells the browser that JavaScript code should
   not be allowed to access the session data. JavaScript on clients is a
-  dangerous thing, ensuring your cookies that contain identity information are
-  safe is always of top importance.
-- `secret`: this option should be a long random string that you create. It
-  should be the same across all your webservers, but never shared publicly or
+  dangerous thing. Ensuring your cookies that contain identity information are
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
  dangerous thing. It's essential to ensure the safety of cookies that contain identity information.
```
this sentence is super convoluted, if you're going to edit, let's remove passive voice and awkward syntax.

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 7]
```diff
@@ -335,10 +290,10 @@ now are the following two:
...
+  safe is always of utmost importance.
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion

```
see above

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 8]
```diff
@@ -497,6 +451,6 @@ to OIDC which I strongly recommend you read if you're interested in learning
 more about OIDC. You can [check it out
 here](/blog/2017/07/25/oidc-primer-part-1).
 
-You can also [follow myself](https://twitter.com/rdegges) and
+You can also [follow me](https://twitter.com/rdegges) and
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
You can also [follow OktaDev](https://twitter.com/oktadev) and
```
do we need to keep linking to an advocate who's no longer working at Okta? i'm not sure I would... but feel free to disregard this comment...

File: _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md [hunk 9]
```diff
@@ -497,6 +451,6 @@ to OIDC which I strongly recommend you read if you're interested in learning
...
 [Okta](https://twitter.com/oktadev) on Twitter to see more of what I'm working
```
Comment (havi) [inline _source/_posts/2017-10-19-use-openid-connect-to-build-a-simple-node-website.md]:
```suggestion
[Okta](https://twitter.com/oktadev) on Twitter to see more of what we're working on at Okta or follow [Randall](https://twitter.com/rdegges) for my latest work. 
```
here's an equitable workaround

**Review by havi — APPROVED** [review]
Thanks for the updates

**Pull request merged**